### DISCUSSION TITLE
Is it possible to cheat to get coins through fast plot technology?

### DISCUSSION THREAD

**Discussion by benqwu** [Chia-Network/chia-blockchain#6709] [Support]
If someone can use challenge hash as input and plot a file in a very short period of time (such as using GPU for plot, less than 30s), can he get coins?

**Comment by loppefaaret** [comment]
technically, yes - if you can create a on the fly plot that bypasses the plotfilter, every 10 seconds, and the netstpace is 20EiB, you should average a win every 3 months or so... netspace being 22EiB+ today, would make that around 100 days~
turn that around, and actually do it the way it was meant, if you can spew out a plot every 10 seconds, for 100 days, that would equal to 864k k32 plots, or a tad over 83PiB... which would earn you around 1000 XCH a month with a netspace around 22EiB

**Comment by BrandtH22** [comment]
@benqwu , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
